Misha/Text/OB-179_count_vowels

krassovki opened the pull request:
- Added count_vowels_main
- Added count_vowels_test

krassovki committed:
OB-179_count_vowels

- Added count_vowels_main
- Added count_vowels_test

glock3 commented on Misha/Text/OB-179_count_vowels/count_vowels_main.py:
```diff
@@ -0,0 +1,20 @@
+        raise TypeError('TypeError')
+    else:
+        text = text.lower()
+        counted_vowels = {'a': 0, 'e': 0, 'i': 0, 'o': 0, 'u': 0}
```
Good decision! It's correct to use map/dictionary in such kind of tasks!

glock3 commented on Misha/Text/OB-179_count_vowels/count_vowels_test.py:
```diff
@@ -0,0 +1,47 @@
+
+    def test_ccount_vowels_with_int_value(self):
+        with self.assertRaises(TypeError) as context:
+            self.assertRaises(TypeError, count_vowels(123))
```
Disagree. It's 0 vowels in this string

glock3 commented on Misha/Text/OB-179_count_vowels/count_vowels_main.py:
```diff
@@ -0,0 +1,20 @@
+def count_vowels(text):
```
Can you add function that will count overall value of vowels? 
It will get {'a': 2, 'e': 5, 'i': 6, 'o': 4, 'u': 2} and return 19

krassovki committed:
Misha/Text/OB-179_count_vowels

- Modified main
- Modified tests

krassovki committed:
OB-179_count_vowels

- Added summarising of vowels (tests passed)

krassovki committed:
OB-179_count_vowels

- Corrected previous commit, because it was rather empty (i havent saved
  files)
